src/EventListener/WriteListener.php

```diff
@@ -50,7 +50,10 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             case 'PUT':
             case 'PATCH':
             case 'POST':
-                $this->dataPersister->persist($controllerResult);
+                $event->setControllerResult(
```

Comment:
CS: We usually don't split into multiple lines like this... Just keep it in one line.